mmv1/third_party/terraform/website/docs/d/apphub_discovered_workload.html.markdown

```diff
@@ -0,0 +1,50 @@
+---
+subcategory: "Apphub"
+description: |-
+  Get information about a discovered workload.
+---
+
+# google\_apphub\_discovered_workload
+
+Get information about a discovered workload from its uri.
+
+
+## Example Usage
+
+
+```hcl
+data "google_apphub_discovered_workload" "my-workload" {
+  location = "us-central1"
+  workload_uri = "my-workload-uri"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `project` - The host project of the discovered workload.
+* `workload_uri` - (Required)  The uri of the workload (instance group managed by the Managed Instance Group). Example: "//compute.googleapis.com/projects/1/regions/us-east1/instanceGroups/id1"
```

Comment:
Can you change it to `instance group managed by the Instance Group Manager`? Thanks.

Reply:
Done. Thanks!